test/Streamly/Test/Data/Unfold.hs

```diff
@@ -449,6 +450,17 @@ enumerateFromToFractional =
 -- Stream transformation
 -------------------------------------------------------------------------------
 
+scan :: Property
+scan =
+    property
+        $ \f _list ->
+              let _fA = apply f :: Int -> Int
+                  _x = _list :: [Int]
```

Comment:
This looks ugly, why is `_list` even being passed as argument? Why is `_fA` required.